src/test/run-make/cabi-int-widening/cfoo.c

```diff
@@ -0,0 +1,4 @@
+// ignore-license
+int foo(char x) {
```

Comment:
I think `char` is signed already. At least, clang specifies that they're `signext`.